iso690-numeric-brackets-cs.csl

```diff
@@ -35,9 +35,9 @@
   </macro>
   <macro name="editor">
     <names variable="editor">
-      <name and="text" name-as-sort-order="first" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
-        <name-part name="given"/>
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter=", " delimiter-precedes-last="never">
         <name-part name="family" text-case="uppercase"/>
+        <name-part name="given"/>
```

Comment:
could we actually take all the ```<name-part name="give"/>``` out of this style? They don't actually do anything. cs:name-part is only used for styling (e.g. the uppercase used for family names here). It has no effect on either the presence of name parts or their order.